src/Handlebars.Net.Helpers/Helpers/StringHelpers.cs

```diff
@@ -412,6 +412,24 @@ public bool IsNotNullOrEmpty(string value)
         return !string.IsNullOrEmpty(value);
     }
 
+    [HandlebarsWriter(WriterType.String)]
+    public string Substring(string value, int start, int end = default)

```

Comment:
Instead of using `int end = default` can you try to use `int? end = null` ?
(There are some other examples in my code what uses nullable as optional parameter.)

Reply:
Done